src/main/java/racing/view/InputView.java

```diff
@@ -0,0 +1,31 @@
+package racing.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static final String ENTER_TRYCOUNT = "시도할 횟수는 몇회인가요?";
+    private static final String ENTER_NAMES = "경주할 자동차 이름을 입력하세요(이름을 쉼표(,)를 기준으로 구분).";
+    private final Scanner scanner = new Scanner(System.in);
```

Comment:
```suggestion
    private static final String ENTER_TRYCOUNT = "시도할 횟수는 몇회인가요?";
    private static final String ENTER_NAMES = "경주할 자동차 이름을 입력하세요(이름을 쉼표(,)를 기준으로 구분).";
    
    private final Scanner scanner = new Scanner(System.in);
```
여기도 한 줄 띄워주는 것도 괜찮겠네요!